lib/snapshot/snapshot.go

```diff
@@ -12,10 +13,18 @@ import (
 	"sync/atomic"
 	"time"
 
+	"github.com/VictoriaMetrics/VictoriaMetrics/app/vmalert/utils"
 	"github.com/VictoriaMetrics/VictoriaMetrics/lib/logger"
 )
 
-var snapshotNameRegexp = regexp.MustCompile(`^[0-9]{14}-[0-9A-Fa-f]+$`)
+var (
+	snapshotNameRegexp    = regexp.MustCompile(`^[0-9]{14}-[0-9A-Fa-f]+$`)
+	tlsInsecureSkipVerify = flag.Bool("snapshot.tlsInsecureSkipVerify", false, "Whether to skip tls verification when connecting to -snapshotCreateURL")
```

Comment:
Could you please update https://github.com/VictoriaMetrics/VictoriaMetrics/blob/master/docs/vmalert.md#flags section of documentation with the new flags added?

Reply:
did you mean vmbackup documentation instead?

Reply:
Yes, that's correct. Sorry for confusion :)